.gitattributes

```diff
@@ -1,2 +1,3 @@
+Dockerfile* linguist-language=Dockerfile
```

Comment:
Does it have a pattern to do the reverse? (thinking of repositories having a `Dockerfile.dockerignore`)